test/sqllogictest/regex.slt

```diff
@@ -125,192 +125,226 @@ CREATE TABLE t(s string, like_pat string, regex_pat string);
 statement ok
 INSERT INTO t VALUES ('abc', 'a%', 'a.*'), ('ABC', 'a%', 'a.*'), ('ccc', 'a%', 'a.*');
 
+statement ok
+INSERT INTO t VALUES (
+  E'test line 1\ntest line 2',
+  '%1%2%',
+  '1.*2'
+);
+
+# Note that sqllogictest can't handle newlines in string output, so replace those when displaying `s`
```

Comment:
All the tests where I added the `replace` are querying the same table, which has newlines.

Reply:
Right, my proposal is to not add the newline row to the table in the first place. You only want to verify that `.` matches newlines, so can't you just add a couple statements at the end that do this? Does putting this string into the table improve our test coverage otherwise?

Reply:
These queries are testing various ways to invoke the regex machinery, and I'd say it's good to test that all these ways default to matching newlines.

Reply:
E.g.,
- const folding vs dynamic
- `MirScalarExpr::reduce` doing partial const folding on the first argument
- various functions and operators
- various flags given